### ISSUE TITLE
introspect error `We assume an internally valid datamodel before mutating.`

### ISSUE THREAD

**Issue by iknowahra** [prisma/prisma#5786]
Hi Prisma Team! Prisma Migrate just crashed. 

## Command

`introspect`

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v10.13.0           | 
| Prisma CLI  | 2.17.0             | 
| Binary      | 3c463ebd78b1d21d8fdacdd27899e280cf686223| 

## Error
```
Error: [libs/datamodel/connectors/dml/src/model.rs:166:14] We assume an internally valid datamodel before mutating.
```

**Comment by Jolg42** [comment]
Could you share your current `schema.prisma`?

**Comment by tomhoule** [comment]
We changed a lot of this logic — can you try again with 2.18.0 once it's released?

**Comment by do4gr** [comment]
Hey, thanks for reporting this. 

Unfortunately we were not collecting the information in our error reporting in a format that allows us to replay the bug. That has changed with 2.19 which should be out next Tuesday. Can you please try again then? This should allow us to exactly pinpoint the issue. 

I'm closing this one now because we cannot act on it (due to our, not your fault). Please just open a new issue once you retried with 2.19 and still get an error.

**Comment by Jolg42** [comment]
@iknowahra Note: it's also available in our dev version `2.19.0-dev.27` if you want to try it before the official 2.19 release
